server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectionAlgorithm.java

```diff
@@ -216,6 +225,46 @@ private long removeBlipCandidates(GarbageCollectionEnvironment gce,
     return blipCount;
   }
 
+  @VisibleForTesting
+  /**
+   *
+   */
```

Comment:
Should put something here...

```suggestion
  /**
   * Double check no tables were missed during GC.
   */
```

Reply:
Addressed in c4e0ab918bf002c83d4200af67df0578aff44e73